doc/contributing/backporting-to-release-lines.md

```diff
@@ -2,136 +2,103 @@
 
 ## Staging branches
 
-Each release line has a staging branch that the releaser will use as a scratch
-pad while preparing a release. The branch name is formatted as follows:
-`vN.x-staging` where `N` is the major release number.
+Each release line has a staging branch that serves as a workspace for preparing releases.
+The branch format is `vN.x-staging`, where `N` is the major release number.
 
-For the active staging branches see the [Release Schedule][].
+For active staging branches, refer to the [Release Schedule][].
 
-## What needs to be backported?
+## Identifying backport needs
```

Comment:
Why this change?

Reply:
According to the Microsoft Style Guide:
"Keep headings as short as possible, and put the most important idea at the beginning. This is especially critical in blogs and social media."

The last heading had the most important idea (backporting) at the end, and this one has it in the middle. Additionally, this heading, not being a question, is shorter.

Reply:
I disagree, the most important is "What": the whole document deals with backporting, this specific section is about "what" needs to be.

Reply:
I've updated the title, better? If not, any suggestions?